packages/graph-editor/src/components/flow/wrapper/base.tsx

```diff
@@ -91,11 +90,9 @@ export const BaseNodeWrapper = (props: NodeProps) => {
             >
               <Stack direction="row" gap={2} align="center">
                 {icon && (
-                  <IconoirProvider
-                    iconProps={{ width: '0.875em', height: '0.875em' }}
-                  >
+                  <div style={{ width: '0.875em', height: '0.875em' }}>
```

Comment:
@AlexBxl you can remove this div entirely, the icons have the width 16 and height 16 by default